stdlib/REPL/src/REPL.jl

```diff
@@ -1448,8 +1448,8 @@ function capture_result(n::Ref{Int}, @nospecialize(x))
     mod = Base.MainInclude
     if !isdefined(mod, :Out)
         @eval mod global Out
-        @eval mod export Out
         setglobal!(mod, :Out, Dict{Int, Any}())
+        @eval Main using Base.MainInclude: Out
```

Comment:
Is the REPL using Main here? I thought that was now configured at runtime

Reply:
Yes the REPL might be using another module, but I don't think we should add names to modules other than Main.

Reply:
Counter-point though: it seems pretty odd to explicitly add them to Main, when the user is not operating in Main

Reply:
Adding them to some other module, like a package you're working on, seems even worse. In any case this is similar to what we did before --- we only added it to `MainInclude`.